trusted-types/trusted-types-reporting.html

```diff
@@ -6,6 +6,11 @@
   <script src="/content-security-policy/support/testharness-helper.js"></script>
 </head>
 <body>
+  <!-- Some elements for the tests to act on. -->
```

Comment:
Did this cause a problem when running them? LGTM either way, just trying to understand the why?

Reply:
Same here, I noticed some flakiness in firefox after implementing svg href due to that. actually it seems there used to be a flakiness in chromium too (will remove that comment)